### ISSUE TITLE
Kill switch disabled

### ISSUE THREAD

**Issue by danielehrhardt** [mullvad/mullvadvpn-app#6369]
### Is it a bug?

- [X] I know this is an issue with the app, and contacting Mullvad support is not relevant.

### I have checked if others have reported this already

- [X] I have checked the issue tracker to see if others have reported similar issues.

### Current Behavior

I have an issue with the Mullvad macOS app where the kill switch is enabled and cannot be disabled. This has been previously discussed in the following issues:
https://github.com/mullvad/mullvadvpn-app/issues/3430
https://github.com/mullvad/mullvadvpn-app/issues/4069

### Expected Behavior

The kill switch option should be usable, allowing users to enable or disable it according to their preferences.

### Steps to Reproduce

1. Open the Mullvad macOS app.
2. Attempt to disable the kill switch option in the settings.
3. Notice that the option to disable the kill switch is disabled.

### Operating system version

macOS 14.5

### Mullvad VPN app version

2024.3

### Additional Information

**Problem Explanation**
When the kill switch is enabled and the VPN is unable to establish a connection, the entire internet is blocked, even if the internet connection without the VPN is functioning correctly. To restore internet access, I have to start the Mullvad app and manually click "Disconnect."

This behavior is problematic because there are various scenarios where the VPN might fail to connect, and it results in the internet being completely inaccessible. For users like myself who do not require a kill switch, this feature is more of a hindrance than a benefit.

**Request**
Please make the kill switch option configurable so that users can choose to disable it if they wish. While it is fine to have it enabled by default, users should have the ability to change this setting based on their needs.

Thank you for your attention to this matter.

**Comment by MrChocolatine** [comment]
> What is the latest stable Mullvad available version for Mac? My Mac is stopped by one year. Thanks.
@serrq 

https://github.com/mullvad/mullvadvpn-app/releases

**Comment by ZisisKostakakis** [comment]
@MarkusPettersson98 

I have currently been using Mullvad VPN for the past couple of months and had zero issues, disconnecting the VPN and continuing to use the internet. 

For the past couple of days, it seems that the kill switch is now giving me trouble (Working as expected I guess). I would like the option of using the Mullvad VPN whenever I want to and also being able to disconnect from it and continue to use the internet.

I am not looking for split tunneling but rather the option to continue using the internet by default when the VPN is off.

Some websites block the Mullvad addresses like Reddit, and I am also blocked by my work-related servers due to it. 

I would like privacy when I want to have it. Is there any option for this? Am I misusing the VPN in some sort of unknown way? Cause it was working like I wanted it before.

**Comment by MrChocolatine** [comment]
> I would like the option of using the Mullvad VPN whenever I want to and also being able to disconnect from it and continue to use the internet.
> ...
> the option to continue using the internet by default when the VPN is off.

Why don't you just "Disconnect" from the VPN?

You just need to make sure `Lockdown mode` is disabled, when it is enabled it requires you to be connected to the VPN to use internet.

<img width="297" alt="Image" src="https://github.com/user-attachments/assets/59226be0-6fb0-4248-9dfe-8a3d9e435551" />

**Comment by ZisisKostakakis** [comment]
@MrChocolatine 

That's my issue, simply disconnecting from the VPN blocks my internet access.

Lockdown mode is disabled.

**Comment by MrChocolatine** [comment]
> That's my issue, simply disconnecting from the VPN blocks my internet access.
> 
> Lockdown mode is disabled.

Even when the VPN app is closed? (and `Lockdown mode` still disabled)

**Comment by MarkusPettersson98** [comment]
Huh, that seems like a bug. If you disconnect and `Lockdown mode` is disabled the Mullvad VPN app should not prevent you from access the internet. Would you mind sending us logs? You'll find them under Settings > Support > Report a problem > View app logs. They are safe to share as sensitive information (account number, ip addresses) is redacted.

Please open a new issue for this bug specifically. Thanks 💛

**Comment by MarkusPettersson98** [comment]
@danielehrhardt Did you see this comment? https://github.com/mullvad/mullvadvpn-app/issues/7516#issuecomment-2614553027 :blush:

**Comment by Mrs-Skelli** [comment]
Hi! I'm experiencing the same issue at the moment. Would be happy to send my logs.

**Comment by BoscoDomingo** [comment]
Same problem happening today. Lockdown mode completely disabled, and yet disconnecting from VPN cuts all internet access

Edit: Since it is off-topic, I'll simply edit the comment. The way I solved this was switch to my phone's hotspot, then back to my Wi-Fi. This allowed me to turn off the VPN without blocking traffic. I already sent logs to the team over email

**Comment by MarkusPettersson98** [comment]
@BoscoDomingo Would you mind opening a new issue and attach logs? I feel like this is the wrong place to discuss your problem(s), since the original issue is ultimately about UX :blush:

**Comment by tam5070-ux** [comment]
> [@BoscoDomingo](https://github.com/BoscoDomingo) Would you mind opening a new issue and attach logs? I feel like this is the wrong place to discuss your problem(s), since the original issue is ultimately about UX 😊

I'm having the same problem as the user above. Lockdown mode disabled, but if I disconnect from the VPN or close MullVad - I lose all internet access on my device. Completely unacceptable behavior - I can't even risk uninstalling it completely because then I might just brick this thing.

**Comment by MarkusPettersson98** [comment]
> I'm having the same problem as the user above. Lockdown mode disabled, but if I disconnect from the VPN or close MullVad - I lose all internet access on my device. Completely unacceptable behavior - I can't even risk uninstalling it completely because then I might just brick this thing.

This seems like a bug indeed. To be able to assist you, we would need some way to reproduce your issue, and preferably logs. Feel free to open a new issue providing as much details as you are able to :blush:

**Comment by VictorHermez** [comment]
This seems like a bug indeed. To be able to assist you, we would need some way to reproduce your issue, and preferably logs.


Well, here are my logs if it helps. This is why I stopped using another VPN provider. Really dont want to swap to another one as Mullvad is killer for me in terms of privacy.

[13381fcc-ffe0-41b0-be3c-e56b38608a5b.log](https://github.com/user-attachments/files/21886111/13381fcc-ffe0-41b0-be3c-e56b38608a5b.log)

**Comment by VictorHermez** [comment]
@MarkusPettersson98 
> This seems like a bug indeed. To be able to assist you, we would need some way to reproduce your issue, and preferably logs. Feel free to open a new issue providing as much details as you are able to 😊


Well, here are my logs if it helps. This is why I stopped using another VPN provider. Really dont want to swap to another one as Mullvad is killer for me in terms of privacy.

[13381fcc-ffe0-41b0-be3c-e56b38608a5b.log](https://github.com/user-attachments/files/21886138/13381fcc-ffe0-41b0-be3c-e56b38608a5b.log)

**Comment by etn0m** [comment]
Just want to add that I have this issue too. Have to force shutdown my machine because mullvad keeps locking out the internet connection. Been having this issue for months now.

This typically happens every other morning or so when I wake my machine from sleep. MacOS Sequoia 15.6.1 (24G90)

**Comment by VictorHermez** [comment]
> Just want to add that I have this issue too. Have to force shutdown my machine because mullvad keeps locking out the internet connection. Been having this issue for months now.
> 
> This typically happens every other morning or so when I wake my machine from sleep. MacOS Sequoia 15.6.1 (24G90)


Found a temp workaround. Its not pretty but it works for now till they fix this.
Just set a custom DNS address to Google (8.8.8.8 or 8.8.4.4) or Cloudflare in the meantime and you will get your internet access back when the VPN is off.

**Comment by MarkusPettersson98** [comment]
> > Just want to add that I have this issue too. Have to force shutdown my machine because mullvad keeps locking out the internet connection. Been having this issue for months now.
> > This typically happens every other morning or so when I wake my machine from sleep. MacOS Sequoia 15.6.1 (24G90)
> 
> Found a temp workaround. Its not pretty but it works for now till they fix this. Just set a custom DNS address to Google (8.8.8.8 or 8.8.4.4) or Cloudflare in the meantime and you will get your internet access back when the VPN is off.

That's interesting. Thanks, it could be useful for while debugging.
